HelpAssistHer/help-assist-her
# Do not list

## Description
 - The button is similar to the Out of Business button
 - When the button is clicked, the orange overlay will be over the form so that the form cannot be edited.
 - The button is right below the Out of Business button

## Test Plan
 - Start the app, log in to it
 - Reload the browser
 - Do Not List button will appear below the out of business button.
 - Hover the button it will turn orange
 - click the button it will turn the form orange and not editable mode
